README.md

```diff
@@ -1 +1 @@
-# ToDoList
```

Comment:
- [ ] **Required:** For have a professionally set up project, a README is required. This gives a summary of the project, especially to recruiters. Kindly include a [REAME using the Microverse Template.](https://github.com/microverseinc/readme-template)